_release/0.17.1.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Apache Arrow 0.17.1 Release
+permalink: /release/0.17.1.html
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Apache Arrow 0.17.1 (18 May 2020)
+
+This is a major release covering more than 1 months of development.
+
+## Download
+
+* [**Source Artifacts**][1]
+* **Binary Artifacts**
+  * [For CentOS][2]
+  * [For Debian][3]
+  * [For Python][4]
+  * [For Ubuntu][5]
+* [Git tag][6]
+
+## Contributors
+
+This release includes 27 commits from 11 distinct contributors.
+
+```console
+$ git shortlog -sn apache-arrow-0.17.0..apache-arrow-0.17.1
+     9	Krisztián Szűcs
+     4	Sutou Kouhei
+     3	Neal Richardson
+     3	Wes McKinney
+     2	Antoine Pitrou
+     1	Eric Erhardt
+     1	Joris Van den Bossche
+     1	Micah Kornfield
+     1	Paul Taylor
+     1	Yibo Cai
+     1	Yuan Zhou
+```
+
+## Patch Committers
```

Comment:
Delete this?

Reply:
Bug in changelog.py, already created an issue for it.